firmware/ec/test/Makefile

```diff
@@ -87,9 +87,15 @@ endif
 
 # Make Targets
 # =============================================================================
-test: $(BUILD_PATHS) $(RESULTS)
+
+test: get_cfg ttest
+
+ttest: $(BUILD_PATHS) $(RESULTS)
```

Comment:
I think you could just make test dependent on get_cfg finishing, if you want. i.e.
```
test: $(BUILD_PATHS) $(RESULT) get_cfg
```